### PR TITLE
Change project name and owner in .all-contributorssrc

### LINKED ISSUE #37: Add links to CONTRIBUTING.md
Links are not working in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by aaditkamat**
This fixes #37 so that the the documentation and code contributions can be viewed through the icons from CONTRIBUTORS.md. The design and  tutorial contributions however have in-place reference links (links within the same web page) so those do not link to the respective websites.

### CODE REVIEW

File: Day3/README.md [hunk 1]
```diff
@@ -106,6 +106,11 @@ public class HammingDistance {
  */
  #include<iostream>
  #include<string>
+<<<<<<< HEAD
```
Comment (MadhavBahlMD) [inline Day3/README.md]: Remove all the  `<<<<<<<< HEAD`  and extra lines that were added to help resolve merge conflicts.
No need for them in the code

File: Day3/README.md [hunk 2]
```diff
@@ -106,6 +106,11 @@ public class HammingDistance {
...
+ 
+ using namespace std;
+ 
+=======
```
Comment (MadhavBahlMD) [inline Day3/README.md]: Remove this line

**Comment by MadhavBahlMD** [comment]
Also, keep taking pull frequently to prevent merge conflicts

**Comment by aaditkamat** [comment]
Hi @imkaka has already been added to CONTRIBUTORS.md and when I tested out the links pointed to by the icons they seemed to be working fine.

**Comment by MadhavBahlMD** [comment]
Ok, so you can add @shivank86 he has contributed but he is not in the contributors list

**Comment by MadhavBahlMD** [comment]
LGTM 👍 

Thanks @aaditkamat for the contribution 😃

**Pull request merged**